src/middleware/packages/core/service.js

```diff
@@ -77,6 +73,13 @@ const CoreService = {
             ctx.meta.webId = 'anon';
             return Promise.resolve(null);
           },
+          /**
+           * Call `signature.authorize` and `auth.authorize`
+           * @param {import('./service').MethodAuthenticateContext} ctx
+           * @param {ApiGatewayService.Route} route
+           * @param {IncomingMessage} req
+           * @param {ServerResponse} res
+           */
           authorize(ctx, route, req, res) {
```

Comment:
`authorize` and `authenticate` are two methods defined by the [ApiGateway mixin](https://moleculer.services/docs/0.14/moleculer-web.html#Authorization). But I guess there is no way to guess them from this mixin ?

Reply:
For now, I am not aware of a way to guess the types from that mixin, unfortunately. This seems to be a typescript limitation right now, at least if the service configuration objects are not put in a function call straight away (see https://stackoverflow.com/questions/69976058/is-there-a-way-to-infer-a-generic-type-without-using-a-function).

The type definitions here were rather for demonstration purposes but since this becomes rather bloaty here, I'd probably leave it out?

Reply:
Yes better to leave it out until Moleculer improves its type integration.